sdk/metric/internal/aggregate/aggregate.go

```diff
@@ -88,39 +88,23 @@ func (b Builder[N]) LastValue() (Measure[N], ComputeAggregation) {
 // PrecomputedSum returns a sum aggregate function input and output. The
 // arguments passed to the input are expected to be the precomputed sum values.
 func (b Builder[N]) PrecomputedSum(monotonic bool) (Measure[N], ComputeAggregation) {
-	var s aggregator[N]
+	s := newPrecomputedSum[N](monotonic)
 	switch b.Temporality {
 	case metricdata.DeltaTemporality:
-		s = newPrecomputedDeltaSum[N](monotonic)
+		return b.filter(s.measure), s.delta
```

Comment:
Where does the new `b.filter(` call come from?  Is this fixing a bug, or is this a part of the "bigger picture" change you've prototyped?

Reply:
`filter` filters the input to the measurement if the user has an attribute filter. This is the same as what we were doing with [`input`](https://github.com/open-telemetry/opentelemetry-go/blob/d5d631852a9e565bc6c9dd0f577115e6879df82c/sdk/metric/internal/aggregate/aggregate.go#L56-L67) for and `aggregator`, but it accepts a different function signature to handle the new `Measure` type.

The goal is to unify all aggregators to use `filter` and remove `input`.